cmake/FindZeekSpicy.cmake

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
+#
+# CMake helpers for Zeek plugins to build Spicy parsers.
+#
+# If the full Spicy toolchain is available, either set PATH to contain
+# spicy-config or set SPICY_CONFIG to its location.
+#
+# If only the Spicy runtime is available, set SPICY_RUNTIME_DIR to its
+# installation root. Spicy source files can't be recompiled then.
+
+find_program(spicy_config spicy-config ${SPICY_ROOT_DIR}/bin/spicy-config ${SPICY_ROOT_DIR}/build/bin/spicy-config)
+
+if ( spicy_config )
+    # Full Spicy toolchain available, use it.
+    execute_process(COMMAND "${spicy_config}" --spicyc
+                    OUTPUT_VARIABLE spicyc
+                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+    string(REPLACE "spicyc" "spicyz" spicyz "${spicyc}")
+
+    execute_process(COMMAND "${spicy_config}" --include-dirs --zeek-include-dirs
+                    OUTPUT_VARIABLE spicy_include_directories
+                    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+    string(REPLACE " " ";" spicy_include_directories "${spicy_include_directories}")
+    list(TRANSFORM spicy_include_directories PREPEND "-I")
+    string(REPLACE ";" " " spicy_include_directories "${spicy_include_directories}")
+
+    message(STATUS "Spicy compiler       : ${spicyz}")
+
+elseif ( NOT "${SPICY_RUNTIME_DIR}" STREQUAL "" )
+    # No Spicy toolchain available, but looks like we have the runtime installed.
+    if ( IS_DIRECTORY "${SPICY_RUNTIME_DIR}/runtime/include/spicy" )
+        message(STATUS "No spicyz, cannot recreate Spicy parsers")
```

Comment:
Is this a leftover? We do not check for `spicyz` here.

Reply:
Not a left-over, but a warning that we don't have the toolchain installed. I'll change wording to that, saying `spicyz` is confusing.